### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by RahullRawat**
Added Toast Component

https://ot4kpl.csb.app/components/toast/toast.html

### CODE REVIEW

File: components/toast/toast.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const btnToast = document.querySelector(".btn-toast");
+const toast = document.querySelector(".toast");
+const toastClose = document.querySelector(".toast-close");
+
+const showToast = () => {
+	console.log("show");
```
Comment (Abhijeet43) [inline components/toast/toast.js]: Avoiding putting console logs

File: components/toast/toast.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+						<i class="fab fa-twitter"></i>
+					</a>
+					<a
+						href="https://www.linkedin.com/in/rahul-rawat-1492b71bb/"
+						target="_blank"
+					>
+						<i class="fab fa-linkedin-in"></i>
+					</a>
+				</section>
+			</nav>
```
Comment (Abhijeet43) [inline components/toast/toast.html:39]: indentation could be improved

**Review by Abhijeet43 — APPROVED** [review]
- The functionality is working fine
- Naming conventions are good
- Suggestions: You can improve indentations and avoid using console logs

**Review by Meghana-sk — APPROVED** [review]
Works fine. PR good to merge. However inline styles on iframes can be put in css file(suggestion)

**Comment by RahullRawat** [comment]
> Works fine. PR good to merge. However inline styles on iframes can be put in css file(suggestion)

Sure will work upon this

**Pull request merged**